config/jobs/kubernetes-sigs/cluster-api-provider-vsphere/cluster-api-provider-vsphere-presubmits.yaml

```diff
@@ -1,57 +1,28 @@
 presets:
 - labels:
     preset-cluster-api-provider-vsphere-e2e-creds: "true"
-  env:
-  - name: JUMPHOST
-    valueFrom:
-      secretKeyRef:
-        name: clusterapi-provider-vsphere-ci-prow
-        key: jumphost
-  - name: GOVC_URL
-    valueFrom:
-      secretKeyRef:
-        name: clusterapi-provider-vsphere-ci-prow
-        key: vsphere-server
-  - name: GOVC_USERNAME
-    valueFrom:
-      secretKeyRef:
-        name: clusterapi-provider-vsphere-ci-prow
-        key: vsphere-username
-  - name: GOVC_PASSWORD
-    valueFrom:
-      secretKeyRef:
-        name: clusterapi-provider-vsphere-ci-prow
-        key: vsphere-password
-  - name: TARGET_VM_SSH
-    valueFrom:
-      secretKeyRef:
-        name: clusterapi-provider-vsphere-ci-prow
-        key: target-vm-ssh
-  - name: TARGET_VM_SSH_PUB
-    valueFrom:
-      secretKeyRef:
-        name: clusterapi-provider-vsphere-ci-prow
-        key: target-vm-ssh-pub
   volumeMounts:
-  - name: jumphost-key
-    mountPath: /root/ssh/.jumphost
-  - name: bootstrapper-key
-    mountPath: /root/ssh/.bootstrapper
+  - name: vpn-conf
+    mountPath: /vpn
   volumes:
-  - name: jumphost-key
+  - name: vpn-conf
     secret:
-      secretName: clusterapi-provider-vsphere-ci-prow
+      secretName: cluster-api-provider-vsphere-e2e-config
       defaultMode: 256
       items:
-      - key: jumphost-key
-        path: jumphost-key
-  - name: bootstrapper-key
-    secret:
-      secretName: clusterapi-provider-vsphere-ci-prow
-      defaultMode: 256
-      items:
-      - key: bootstrapper-key
-        path: bootstrapper-key
+      - key: prow.ovpn
+        path: prow.ovpn
+      - key: client.crt
+        path: client.crt
+      - key: client.key
+        path: client.key
+      - key: ca.crt
+        path: ca.crt
+      - key: tls.key
+        path: tls.key
+      - key: envvars.txt
```

Comment:
Hi @yastij,

We removed the envvars from the VPN config secret to use the existing ones.